src/main/java/seedu/address/logic/commands/AddCommand.java

```diff
@@ -22,21 +23,13 @@ public class AddCommand extends Command {
     public static final String COMMAND_WORD = "add";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a person to the address book. "
-            + "Parameters: "
-            + PREFIX_NAME + "NAME "
-            + PREFIX_PHONE + "PHONE "
-            + "[" + PREFIX_EMAIL + "EMAIL] "
-            + "[" + PREFIX_ADDRESS + "ADDRESS] "
-            + "[" + PREFIX_TAG + "TAG]... "
-            + "[" + PREFIX_DATEOFLASTVISIT + "DATEOFLASTVISIT] \n"
-            + "Example: " + COMMAND_WORD + " "
-            + PREFIX_NAME + "John Doe "
-            + PREFIX_PHONE + "98765432 "
-            + PREFIX_EMAIL + "johnd@example.com "
-            + PREFIX_ADDRESS + "311, Clementi Ave 2, #02-25 "
-            + PREFIX_TAG + "friends "
-            + PREFIX_TAG + "owesMoney "
-            + PREFIX_DATEOFLASTVISIT + "01-03-2024";
+            + "Parameters: " + PREFIX_NAME + "NAME " + PREFIX_PHONE + "PHONE " + "[" + PREFIX_EMAIL
+            + "EMAIL] " + "[" + PREFIX_ADDRESS + "ADDRESS] " + "[" + PREFIX_TAG + "TAG]... "
+            + PREFIX_DATEOFLASTVISIT + "DATEOFLASTVISIT " + PREFIX_EMERGENCY_CONTACT + "EMERGENCY_CONTACT \n"
+            + "Example: " + COMMAND_WORD + " " + PREFIX_NAME + "John Doe " + PREFIX_PHONE + "98765432 "
+            + PREFIX_EMAIL + "johnd@example.com " + PREFIX_ADDRESS + "311, Clementi Ave 2, #02-25 "
+            + PREFIX_TAG + "friends " + PREFIX_TAG + "owesMoney " + PREFIX_DATEOFLASTVISIT + "01-03-2024"
```

Comment:
nit: A space could be added behind `01-03-2024 ` so that the result display shows 

`add....d/01-03-2024 ec/92137920 ` instead of `add....d/01-03-2024ec/92137920`

Reply:
good catch, thx!